docs/tutorial/index.md

```diff
@@ -100,185 +74,57 @@ Please export them and delete them. Then proceed with the tutorial and install t
  </details>
 ```
 
+You can now launch WSL instances of Ubuntu on your Windows machine.
 
+### Set up Landscape
```

Comment:
I'd remove this entire section from the tutorial and point to a Landscape tutorial if there is one.
Installing Landscape is unrelated to Ubuntu Pro for WSL.
If really we want to explain how to install Landscape in WSL, it should be a separate tutorial in WSL documentation.

Reply:
@jibel , I agree and the Landscape section currently takes up too much space; however, I would like to leave it here for now.
The purpose of this PR was to streamline the tutorial and improve its style while leaving much of the technical content intact.
There are decisions to be made about restructuring the docs, such as moving material into separate tutorials.
It is better to do this incrementally because otherwise there are too many moving parts.